wordpress-mobile/WordPress-Android
# Merge Release/16.6 back into develop

Merge Release/16.6 (16.6-rc-2 and new release notes) back into develop.

*To Test*
- Check CI is green.
- Make sure `lintVanillaRelease` passes without errors locally.

PR submission checklist:
